tests/beaker_tests/cisco_portchannel_global/test_portchannel_global.rb

```diff
@@ -273,6 +273,42 @@
   },
 }
 
+tests['default_properties_no_resilient'] = {
+  title_pattern:  'default',
+  manifest_props: "
+    bundle_hash                  => 'default',
+    bundle_select                => 'default',
+    symmetry                     => 'default',
+  ",
```

Comment:
This testfile is getting seriously unwieldy. :-(

The only difference between test cases `default_properties_no_rotate` and `default_properties_no_resilient` is that one has a resilient property (based on hardware support) and the boolean value is different for symmetry. This could be simplified a lot if you convert the testfile to use `unsupported_properties` and `test_harness_run`; in which case you'll have one testcase for `_no_rotate` that automatically removes the unsupported properties.